src/mmw/js/src/app.js

```diff
@@ -50,6 +50,19 @@ var App = new Marionette.Application({
         // new project.
         this.currentProject = null;
 
+        $(window)
+            .on('shutterbug-saycheese', function() {
+                // Set fixed width before screenshot to constrain width to viewport
+                $('#model-output-wrapper, .map-container').css({
+                    'width': window.innerWidth
+                });
+            })
+            .on('shutterbug-asyouwere', function() {
+                // Reset after screenshot has been taken
+                $('#model-output-wrapper, .map-container').css({
+                    'width': ''
+                });
+            });
```

Comment:
Originally I tried to put all the Shutterbug code in [`setup.js`](https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/core/setup.js), but I was also looking at #971 and realized that we might need these handlers to be able to access `App.getLeafletMap()`, so it can't be initialized before `App`. Thus I just added it here.

Extracting it into an initialization function within this file sounds like a good compromise. I'll do that.